site/docs/.vuepress/components/webhook-util/translations/en_US.ts

```diff
@@ -0,0 +1,72 @@
+import { Translation } from "./types";
+
+export const lang: Translation = {
+  tokenCard: {
+    disclaimer: {
+      title: "About your token",
+      content:
+        "We never store your bot's token in the backend. It is stored in your browser's localStorage. Do not go around giving your token to random websites, no matter how nicely they ask!",
```

Comment:
```suggestion
        "We don’t take your bot token to the back end. It will only be stored in your browser’s local storage. Don’t provide your bot token to places you don’t trust.",
```